terraform/cloudfoundry/network_acls_cf_in.tf

```diff
@@ -123,7 +123,7 @@ resource "aws_network_acl_rule" "100_ssh" {
     rule_number = 100
     rule_action = "allow"
     from_port = 22
-    to_port = 22
+    to_port = 9300
```

Comment:
Hm, you are creating `118_ingestor_syslog` as well that allows TCP traffic from infra on ports 2514-5514. We can extend this to be 2514-9300. It's still terrible but a little bit less.